### PR TITLE
Pioneer 15 WFP cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
New Cal csvs for Pioneer 15 WFPs
Incoming Inspection document for reference - 3102-00100-00091.docx
*note - need to add the 3 new PARADK sensors to sensor bulk load csv*

WFP 13103-01 
CGINS-DOFSTK-02708__20200801.csv
CGINS-FLORTK-01117__20200728.csv
CGINS-PARADK-20610__20200609.csv

WFP 14300-01
CGINS-DOFSTK-03506__20200717.csv
CGINS-FLORTK-01537__20200716.csv
CGINS-PARADK-20608__20200609.csv

WFP 14300-02
CGINS-DOFSTK-03523__20200716.csv
CGINS-FLORTK-01538__20200716.csv
CGINS-PARADK-20609__20200609.csv

### CODE REVIEW

File: calibration/DOFSTK/CGINS-DOFSTK-03523__20200716.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-3523,CC_frequency_offset,-8.157175e+002,source_file DOFST-K_SBE43F_SN_3523_Calibration_2020-07-16.zip > 3523.cal
+43-3523,CC_oxygen_signal_slope,3.4937e-004,use Soc(adj) on SOCAdjusted pdf
+43-3523,CC_residual_temperature_correction_factor_a,-3.159971e-003,
+43-3523,CC_residual_temperature_correction_factor_b,1.498717e-004,
+43-3523,CC_residual_temperature_correction_factor_c,-2.411948e-006,
+43-3523,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (dbogorff) [inline calibration/DOFSTK/CGINS-DOFSTK-03523__20200716.csv:7]: Looks good!

File: calibration/DOFSTK/CGINS-DOFSTK-03506__20200717.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-3506,CC_frequency_offset,-8.660988e+002,source_file DOFST-K_SBE43F_SN_3506_Calibration_2020-07-17.zip > 3506.cal
+43-3506,CC_oxygen_signal_slope,3.2857e-004,use Soc(adj) on SOCAdjusted pdf
+43-3506,CC_residual_temperature_correction_factor_a,-1.831617e-003,
+43-3506,CC_residual_temperature_correction_factor_b,1.430734e-004,
+43-3506,CC_residual_temperature_correction_factor_c,-1.663473e-006,
+43-3506,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (dbogorff) [inline calibration/DOFSTK/CGINS-DOFSTK-03506__20200717.csv:7]: Looks good!

File: calibration/FLORTK/CGINS-FLORTK-01117__20200728.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+1117,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1117_Calibration_2020-07-28.zip > BBFL2-1117 Charsheets.pdf
+1117,CC_dark_counts_cdom,47,
+1117,CC_dark_counts_chlorophyll_a,48,
+1117,CC_dark_counts_volume_scatter,49,
+1117,CC_depolarization_ratio,0.039,
+1117,CC_measurement_wavelength,700,
+1117,CC_scale_factor_cdom,0.0503,
+1117,CC_scale_factor_chlorophyll_a,0.0108,
+1117,CC_scale_factor_volume_scatter,3.699E-06,
+1117,CC_scattering_angle,124,
```
Comment (dbogorff) [inline calibration/FLORTK/CGINS-FLORTK-01117__20200728.csv:11]: Looks good!

File: calibration/FLORTK/CGINS-FLORTK-01537__20200716.csv [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+1537,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1537_Calibration_2020-07-16.zip > BBFL2-1537 Charsheets.pdf
+1537,CC_dark_counts_cdom,49,
+1537,CC_dark_counts_chlorophyll_a,48,
+1537,CC_dark_counts_volume_scatter,49,
+1537,CC_depolarization_ratio,0.039,
+1537,CC_measurement_wavelength,700,
+1537,CC_scale_factor_cdom,0.0727,
+1537,CC_scale_factor_chlorophyll_a,0.0119,
+1537,CC_scale_factor_volume_scatter,3.617E-06,
+1537,CC_scattering_angle,124,
```
Comment (dbogorff) [inline calibration/FLORTK/CGINS-FLORTK-01537__20200716.csv:11]: Looks good!

File: calibration/FLORTK/CGINS-FLORTK-01538__20200716.csv [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+1538,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1538_Calibration_2020-07-16.zip > BBFL2-1538 Charsheets.pdf
+1538,CC_dark_counts_cdom,49,
+1538,CC_dark_counts_chlorophyll_a,48,
+1538,CC_dark_counts_volume_scatter,47,
+1538,CC_depolarization_ratio,0.039,
+1538,CC_measurement_wavelength,700,
+1538,CC_scale_factor_cdom,0.0771,
+1538,CC_scale_factor_chlorophyll_a,0.0120,
+1538,CC_scale_factor_volume_scatter,3.807E-06,
+1538,CC_scattering_angle,124,
```
Comment (dbogorff) [inline calibration/FLORTK/CGINS-FLORTK-01538__20200716.csv:11]: Looks good!

File: calibration/PARADK/CGINS-PARADK-20608__20200609.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20608,CC_dark_offset,1.0,source_file PARAD-K_QSP2200_SN_20608_Calibration_2020-06-09.pdf
+20608,CC_scale_wet,9.68E-18,
```
Comment (dbogorff) [inline calibration/PARADK/CGINS-PARADK-20608__20200609.csv:3]: Looks good!

File: calibration/PARADK/CGINS-PARADK-20609__20200609.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20609,CC_dark_offset,1.0,source_file PARAD-K_QSP2200_SN_20609_Calibration_2020-06-09.pdf
+20609,CC_scale_wet,9.63E-18,
```
Comment (dbogorff) [inline calibration/PARADK/CGINS-PARADK-20609__20200609.csv:3]: Looks Good!

File: calibration/PARADK/CGINS-PARADK-20610__20200609.csv [hunk 8]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20610,CC_dark_offset,1.0,source_file PARAD-K_QSP2200_SN_20610_Calibration_2020-06-09.pdf
+20610,CC_scale_wet,9.65E-18,
```
Comment (dbogorff) [inline calibration/PARADK/CGINS-PARADK-20610__20200609.csv:3]: Looks good!

**Review by dbogorff — APPROVED** [review]

File: calibration/DOFSTK/CGINS-DOFSTK-03506__20200717.csv (hunk 2 above)
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-03506__20200717.csv:7]: Looks good

**Review by cdobs — APPROVED** [review]
Everything looks good

**Pull request merged**